app/routes/planet_routes.py

```diff
@@ -13,8 +43,33 @@ def get_all_planets():
 @planets_bp.get("/<planet_id>")
 def get_one_planet(planet_id):
     planet = validate_planet(planet_id)
-    return planet.to_dict(),200
-    
+    return planet.to_dict()
+
+@planets_bp.get("/count")
+def get_count():
+    query = db.session.query(Planet.name).count()
+    return {"count" : query}
+
+@planets_bp.put("/<planet_id>")
+def update_planet(planet_id):
+    planet = validate_planet(planet_id)
+    request_body = request.get_json()
+
+    planet.name = request_body["name"]
+    planet.description = request_body["description"]
+    planet.galaxy = request_body["galaxy"]
```

Comment:
Similar problems as for creating a record could happen here as well. What might we want to check for, and how might we respond?